src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -140,19 +140,25 @@ void uct_rc_mlx5_iface_common_query(uct_rc_iface_t *iface,
     iface_attr->cap.put.max_bcopy = iface->super.config.seg_size;
     iface_attr->cap.put.min_zcopy = 0;
     iface_attr->cap.put.max_zcopy = uct_ib_iface_port_attr(&iface->super)->max_msg_sz;
+    iface_attr->cap.put.opt_zcopy_align = UCS_SYS_PCI_MAX_PAYLOAD;
```

Comment:
let's move this to uct_rc_iface_query (to avoid code duplication with rc_verbs_common.c)